server/pbench/bin/gold/test-7.8.txt

```diff
@@ -1039,7 +1039,19 @@ len(actions) = 19
         },
         "_type": "pbench-tool-data-iostat"
     }
-]
+]could not convert binary: /var/tmp/pbench-test-server/tmp/uperf__2016-10-06_16:34:03/1-tcp_stream-16384B-32i/sample1/tools-default/flat8_40gb/sar/sar.data
```

Comment:
so i've removed all sar data from uperf results for now. (old ref: https://github.com/distributed-system-analysis/pbench/pull/523#discussion_r106825374). Do you think we should submit another PR for removing this added sadf compatibility feature from the tree? (and subsequently adding set of instructions to go achieve the same?). 

I'm thinking users would be looking all over the place for older sadf versions (in case they're indexing old results) and presence of / pointer to, these binaries is crucial. Although I guess such cases could be rare.